hack/test-cmd.sh

```diff
@@ -264,4 +264,10 @@ openshift ex router --create --credentials="${KUBECONFIG}"
 [ "$(openshift ex router | grep 'service exists')" ]
 echo "ex router: ok"
 
+[ ! "$(openshift ex registry | grep 'does not exist')"]
```

Comment:
Grep fails with an exit code.

----- Original Message -----
> > @@ -264,4 +264,10 @@ openshift ex router --create
> > --credentials="${KUBECONFIG}"
> >  [ "$(openshift ex router | grep 'service exists')" ]
> >  echo "ex router: ok"
> >  
> > +[ ! "$(openshift ex registry | grep 'does not exist')"]
> 
> shouldn't this fail with some reasonable exit code instead of parsing the
> output?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/1100/files#r25151005